### PR TITLE
chore(notifications): add notifications package

### LINKED ISSUE #3565: feature(notifications): create notification manager

### PR DESCRIPTION
**Pull request by michalszorad**
## Description / Change(s) / Related issue(s)

Add notifications package

##### Ticket

[YOMO-1862](https://emurgo.atlassian.net/jira/software/c/projects/YOMO/issues/YOMO-1862)

[YOMO-1862]: https://emurgo.atlassian.net/browse/YOMO-1862?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/notifications/package.json [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+    "preset": "react-native",
+    "setupFiles": [
+      "<rootDir>/jest.setup.js"
+    ]
+  },
+  "dependencies": {
+    "@emurgo/cip14-js": "^3.0.1",
+    "bech32": "^2.0.0",
+    "bip39": "^3.1.0"
+  },
```
Comment (stackchain) [inline packages/notifications/package.json]:
```suggestion
```

File: packages/notifications/package.json [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+    "axios": "^1.5.0",
+    "axios-mock-adapter": "^1.21.5",
+    "commitlint": "^17.0.2",
+    "del-cli": "^5.0.0",
+    "dependency-cruiser": "^13.1.1",
+    "eslint": "^8.4.1",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-ft-flow": "^3.0.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "flowgen": "^1.21.0",
```
Comment (stackchain) [inline packages/notifications/package.json:157]: please remember to copy the folder `script/` it contains the scripts to create the flow types

File: yoroi.code-workspace [hunk 3]
```diff
@@ -71,6 +71,10 @@
     {
       "name": "@yoroi/e2e-wallet-mobile",
       "path": "./e2e/wallet-mobile"
+    },
+    {
+      "name": "@yoroi/notifications",
+      "path": "./packages/notifications"
```
Comment (stackchain) [inline yoroi.code-workspace:77]: please update the root scripts to add packages

**Review by banklesss — APPROVED** [review]

File: yoroi.code-workspace (hunk 3 above)
Comment (michalszorad) [reply yoroi.code-workspace:77]: Updated

File: packages/notifications/package.json (hunk 2 above)
Comment (michalszorad) [reply packages/notifications/package.json:157]: Updated

File: packages/notifications/package.json (hunk 1 above)
Comment (michalszorad) [reply packages/notifications/package.json]: Updated

**Pull request merged**